### PR TITLE
feat: fox page plugin routing (do not merge before 1782)

### PR DESCRIPTION
**Pull request by chxash**
## Description

This implements basic routing for the fox page.
Dependent on https://github.com/shapeshift/web/pull/1782 being merged

## Notice

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/1763

## Testing

- Enable fox page flag
- Browse to fox or fox/fox page => see blank page with "fox tab" text
- Browse to fox/foxy or fox/onefox => see blank page with "foxy tab" text

### CODE REVIEW

**Review by gomesalexandre — COMMENTED** [review]
Added a comment on what looks like early routing logic (as in, internal fox page routing vs. plugin routing, which is what we want to add here) , otherwise tested and looks good.

File: src/plugins/foxPage/foxPage.tsx [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import { RawText } from 'components/Text'
 
-export const FoxPage = () => {
+export enum FoxPageTab {
```
Comment (gomesalexandre) [inline src/plugins/foxPage/foxPage.tsx:3]: This also includes oneFOX

Comment (chxash) [reply src/plugins/foxPage/foxPage.tsx:3]: fixed !

**Review by gomesalexandre — APPROVED** [review]
Tested with latest diff, LGTM, all three routes are working (tested by checking console logs)

**Comment by elmutt** [comment]
> ## Description
> This implements basic routing for the fox page. Dependent on #1782 being merged
> 
> ## Notice
> * [x]  Have you followed the guidelines in our Contributing guide?
> * [x]  Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
> 
> ## Pull Request Type
> * [ ]  🐛 Bug fix (Non-breaking Change: Fixes an issue)
> * [ ]  🛠️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
> * [x]  💅 New Feature (Breaking/Non-breaking Change)
> 
> ## Issue (if applicable)
> closes #1763
> 
> ## Risk
> ## Testing
> * Enable fox page flag
> * Browse to fox or fox/fox page => see blank page with "fox tab" text
> * Browse to fox/foxy or fox/onefox => see blank page with "foxy tab" text
> 
> ## Screenshots (if applicable)

maybe make it a draft until it should be merged?

**Review by 0xdef1cafe — APPROVED** [review]
fixed conficts from your previous PR, tested and working, ty

**Pull request merged**